### ISSUE TITLE
md-tab: Dummy tabs not properly hidden from a screen reader [Accessibility]

### ISSUE THREAD

**Issue by KristenLouca** [angular/material#10075]
**Actual Behavior**:
 -  `What is the issue? *`
When using md-tab, the dummy tabs that are visually hidden with the "md-visually-hidden" class are still visible and accessible by a screen reader (JAWS). This results in 2 mirror copies of the list of tabs appearing when using arrow-keys to browse the page.
 -  `What is the expected behavior?`
Dummy tabs, if visually hidden, should be hidden from a screen reader as well using the 'aria-hidden' attribute. Only one set of tabs should be visible by a screen reader when using the arrow-keys to browse.
**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:`
N/A (issue present in current demos on Angular Material website)
 -  `Details:`
Here is the offending <div> that shows up in the code when the md-tabs are rendered:
<div class="_md-visually-hidden md-dummy-wrapper"> 

This should also have the attribute of 'aria-hidden="true"'. I confirmed that adding this attribute to this particular <div> wrapper fixes the issue.
**Angular Versions**: *
 -  `Angular Version:`
1.5.3
 -  `Angular Material Version:`
1.1.0-rc2
**Additional Information**:
 -  `Browser Type: *`
Firefox and IE
 -  `Browser Version: *`
Firefox latest version and IE11
 -  `OS: *`
Windows 7/64 bit
 -  `Stack Traces:`
N/A
----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by KristenLouca** [comment]
Fix to above issue - the div I am speaking of that needs the aria-hidden is the wrapper for the dummy tabs. Totally forgot that the HTML would likely be stripped out.
